tests/travis/install.sh

```diff
@@ -8,6 +8,16 @@ sudo apt-get install -qq --force-yes libestr-dev librelp-dev libfastjson-dev lib
 	libcurl4-gnutls-dev
 sudo apt-get install -qq python-docutils
 
+if [ "$DISTRIB_CODENAME" == "trusty" ]; then
+	set -ex
+	WANT_MAXMIND=1.2.0
+	curl -Ls https://github.com/maxmind/libmaxminddb/releases/download/${WANT_MAXMIND}/libmaxminddb-${WANT_MAXMIND}.tar.gz | tar -xz
+	(cd libmaxminddb-${WANT_MAXMIND} ; ./configure --prefix=/usr CC=gcc CFLAGS="-Wall -Wextra -g -pipe -std=gnu99" ; sudo make install)
+	set +x
+else
+	sudo apt-get install -qq libmaxminddb-dev
```

Comment:
We don't need to test on all platforms. If it works on newer than trusty, we can simply use that and disable the tests on trusty. We do this with some other tests as well. It's also a question of CI runtime, which already is close to the limit. Usually, the functionality tests work well on all platforms if it works on one. Of course not always, but we need a compromise between CI coverage and runtime.